### PR TITLE
refactor: output model now does not have address anymore, it has a buffer script that can be decoded

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

File: src/wallet/sendTransaction.ts [hunk 1]
```diff
@@ -166,8 +167,10 @@ class SendTransactionWalletService extends EventEmitter {
         throw new SendTxError(`Address ${o.address} is not valid.`);
       }
       const tokenData = (o.token in tokens) ? tokens.indexOf(o.token) + 1 : 0;
-      const outputOptions = { tokenData, timelock: o.timelock || null };
-      outputsObj.push(new Output(o.value, address, outputOptions));
+      const outputOptions = { tokenData };
+      const p2pkh = new P2PKH(address, { timelock: o.timelock || null });
+      const p2pkhScript = p2pkh.createScript()
+      outputsObj.push(new Output(o.value, p2pkhScript, outputOptions));
```
Comment (msbrogli) [inline src/wallet/sendTransaction.ts:173]: Maybe `p2pkh.createOutput()`?

**Review by msbrogli — APPROVED** [review]
Well done! \o/

File: src/wallet/sendTransaction.ts (hunk 1 above)
Comment (pedroferreira1) [reply src/wallet/sendTransaction.ts:173]: Do you mean to create a method in P2PKH class that receives value and tokenData and creates an output from a P2PKH object?

Comment (pedroferreira1) [reply src/wallet/sendTransaction.ts:173]: As we've discussed in a call there is no need to create this method, it wouldn't help much.

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**